packages/runner/injection/patches/cookies.ts

```diff
@@ -0,0 +1,133 @@
+import { AutomationCookie } from '@packages/server/lib/automation/cookies'
+import {
+  CookieJar,
+  toughCookieToAutomationCookie,
+  automationCookieToToughCookie,
+} from '@packages/server/lib/util/cookies'
+import { Cookie as ToughCookie } from 'tough-cookie'
+
+const parseDocumentCookieString = (documentCookieString: string): AutomationCookie[] => {
+  if (!documentCookieString || !documentCookieString.trim().length) return []
+
+  return documentCookieString.split(';').map((cookieString) => {
+    const [name, value] = cookieString.split('=')
+
+    return {
+      name: name.trim(),
+      value: value.trim(),
+      domain: location.hostname,
+      expiry: null,
+      httpOnly: false,
+      maxAge: null,
+      path: null,
+      sameSite: 'lax',
```

Comment:
This is done synchronously to seed the cookie jar with the simulated cookies on the initial response? I think ideally we could get these cookies out of CDP and set them here, but we need to do that async, and the only thing I think we get out of it is accurate `path`, `sameSite`, etc... I think these should be sane defaults, right?

Reply:
Right, it's to grab any value currently in document.cookie so if a user reads it, it's still there. If it's originally read from `document.cookie`, its path/sameSite/etc must match already, so it makes sense for it to be read from `document.cookie` subsequently, right?

Reply:
I think so. Does this get updated to the known values once the automation cookies sync in?

Reply:
It no longer polls to sync with the automation cookies, since that leads to race conditions and was pretty non-deterministic.

Reply:
oh duh 🤦🏻 . I wonder in the future if we can get automation cookies w/ simulated cookies to know these values upon injection. Otherwise though I feel like the defaults are sane and fine.